erpnext/manufacturing/report/bom_stock_report/bom_stock_report.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2013, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+import frappe
+from frappe import _
+
+
+def execute(filters=None):
+    if not filters: filters = {}
+    columns = get_columns()
+    bom_stock = get_bom_stock(filters)
+    data = []
+
+    for row in bom_stock:
```

Comment:
@creamdory,

No need to iterate over bom_stock, just remove the as_dict=1 from `get_bom_stock` and directly assign the result to data